packages/core/src/services/workspaces/usage.ts

```diff
@@ -36,9 +37,8 @@ export async function computeWorkspaceUsage(
     db,
   )
 
-  const createdAtDate = workspace.createdAt
+  const createdAtDate = workspace.currentSubscription.createdAt
```

Comment:
this can't be merged atm because we would be effectively resetting the credit count for all users.

Reply:
Either create a migration to set the default subscription.created_at date for each workspace the same as the workspace.created_at,

or just merge it and some people will be happy about it xd

Reply:
What about a data migration that sets subscription.created_at = workspace.created_at?